src/helpers/github.ts

```diff
@@ -99,3 +99,66 @@ export async function isCiGreen({ octokit, logger, env }: Context, sha: string,
     return false;
   }
 }
+
+function parseTarget({ payload }: Context, target: string) {
+  const owner = payload.repository.owner?.login || "";
+  const [orgParsed, repoParsed] = target.split("/");
```

Comment:
Should we check length here otherwise `orgPassed` might be the repo name?

Reply:
It doesn't make sense to return an empty string. Instead it should throw an error (or be `null`)

Reply:
A repo name requires the organization to be set. This is to avoid mistakes of multiple organizations having the same repo names. I added a throw if no owner is found.